src/hooks/prisma.ts

```diff
@@ -59,9 +83,7 @@ function getFunctionInfo(model: string, action: string, moduleId: string) {
   const key = model + "." + action;
 
   if (!functionInfos.has(key)) {
-    const params = ["data", "include", "where"].map((k) => {
-      return { type: "Identifier", name: k } as ESTree.Identifier;
-    });
+    const params = [{ type: "Identifier", name: "args" } as ESTree.Identifier];
```

Comment:
Not sure why this change is necessary. Nb, you can use `generate.identifier()`.

Reply:
There can be lots of options like `data, select, include, where, orderBy, take, skip, distinct, cursor, relationLoadStrategy,` etc. Having a parameter for each one seeme not to look good when exploring the AppMap in the viewer, since most of them are irrelevant in a specific call and end up showing as "undefined." 

Making different subsets of parameters for each Prisma model query isn't practical either, because, for example, `findMany` can have around 10 options and many or them will again be "undefined" in a specific call.

Reply:
Makes sense, thanks.